SPECS/kernel-azure/config_aarch64

```diff
@@ -808,6 +808,7 @@ CONFIG_GCC_PLUGINS=y
 # CONFIG_GCC_PLUGIN_CYC_COMPLEXITY is not set
 # CONFIG_GCC_PLUGIN_LATENT_ENTROPY is not set
 # CONFIG_GCC_PLUGIN_RANDSTRUCT is not set
+CONFIG_FUNCTION_ALIGNMENT=0
```

Comment:
Why do we do different function alignment for arm?

Reply:
ARM64 does not explicilty introduce a function alignment where x86_64 does: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/ab8f581408c526319fefbd9db323d43b4342a5fc#diff-b7748a562e1c9f49201f8ecd3f2cfd1e6eb71c83836abccc7e4199f7b4de3de6R270